tinygrad/tensor.py

```diff
@@ -158,7 +158,7 @@ def randn(*shape, **kwargs) -> Tensor: return Tensor(LazyNumpyArray(lambda lna:
   # ***** rng hlops *****
 
   @staticmethod
-  def uniform(*shape, **kwargs) -> Tensor: return Tensor.rand(*shape, **kwargs) * 2 - 1
+  def uniform(*shape, low=-1.0, high=1.0, **kwargs) -> Tensor: return Tensor(LazyNumpyArray(lambda lna: Tensor._rng.uniform(low=low, high=high, size=lna.shape), shape, np.float32), **kwargs)
```

Comment:
Should be good after one more change.

I'm fine with the low and high params, but use Tensor.rand as a base, not another LazyNumpyArray.

Reply:
I see but I think I shouldn't. Because to use `Tensor.rand` as a base, I need to change that function from `Tensor._rng.random` to `Tensor._rng.uniform`. Is it okay?

Reply:
Please look into that more.

Reply:
Hm, because `np.random.Generator.random` does not have low and high params but `np.random.Generator.uniform` does, it is better to use LazyNumpyArray? Am I missing something?

Reply:
Done! ^_^